.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,22 @@
+# This is workflow for spell checking using PySpelling lib (https://pypi.org/project/pyspelling/)
+name: Spellcheck
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    paths: "docs/**"
```

Comment:
imho should be triggered only on PR events

Reply:
Updated.